content/docs/apps/continuous-deployment.md

```diff
@@ -172,7 +157,7 @@ Change `APP` and `DOMAIN` to match your application, and set up the following en
 
 | Name    | Value              |
 |---------|--------------------|
-| CF_API  | East/West: `api.cloud.gov` / GovCloud: `api.fr.cloud.gov`     |
+| CF_API  | GovCloud: `api.fr.cloud.gov`     |
```

Comment:
Do we want to even mention `GovCloud` here? Not leaning a certain way, just wondering.